database/dbbuild.sql

```diff
@@ -0,0 +1,38 @@
+BEGIN;
+
+DROP TABLE IF EXISTS recipe, users cascade;
+
+CREATE TABLE IF NOT EXISTS users (
+  id            SERIAL        PRIMARY KEY,
+  username      VARCHAR(15)   NOT NULL UNIQUE,
+  password      VARCHAR(70)   NOT NULL,
+  name          VARCHAR(20)   DEFAULT NULL,
+  surname       VARCHAR(20)   DEFAULT NULL,
+  email         VARCHAR(30)   DEFAULT NULL UNIQUE
+);
+
+CREATE TABLE IF NOT EXISTS recipe (
+  id            SERIAL        PRIMARY KEY,
+  title         VARCHAR(100)  NOT NULL,
+  difficulty    TEXT          DEFAULT NULL,
+  duration      INTEGER       DEFAULT NULL,
+  ingredients   TEXT          NOT NULL,
+  procedure     TEXT          NOT NULL,
+  cuisine       TEXT          NOT NULL,
+  author_id     INTEGER       REFERENCES users(id) ON UPDATE cascade
+);
+
+INSERT INTO users (username, password, name, surname, email) VALUES
```

Comment:
i hope this users table is not like last project.
what i mean is that i hope it's not limited to those 5 users